### PR TITLE
Tweak: Detached Slides on Scroll control - Core Base [ED-10529]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/scss/editor/panel/controls/_select.scss [hunk 1]
```diff
@@ -39,4 +39,14 @@
 			pointer-events: none;
 		}
 	}
+
+	.elementor-control-field {
+
+		&.elementor-control-field-select-small {
```
Comment (rodolphebertozzo) [inline assets/dev/scss/editor/panel/controls/_select.scss:45]: Is the & really necessary in this case ? 
why not just write: 
```
.elementor-control-field {
  .elementor-control-field-select-small {
    .elementor-control-input-wrapper {
      max-width: 80px;
    }
  }
}
```

**Review by rodolphebertozzo — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/controls/_select.scss (hunk 1 above)
Comment (rodolphebertozzo) [reply assets/dev/scss/editor/panel/controls/_select.scss]: Indeed, it seems to me more coherent

Comment (hein-obox) [reply assets/dev/scss/editor/panel/controls/_select.scss]: Thank you for your feedback, @rodolphebertozzo .

Are you part of the Elementor team or you part of the community? What kind of work do you in the Elementor context? #curious

Comment (rodolphebertozzo) [reply assets/dev/scss/editor/panel/controls/_select.scss]: I'm part of the community in a way and I'm a code contributor to improve the widgets, I'm not a developer, I learn languages by myself, especially SASS, by the way, I'll be glad if you can have a look at this PR ;) https://github.com/elementor/elementor/pull/20906

Comment (hein-obox) [reply assets/dev/scss/editor/panel/controls/_select.scss:45]: @rodolphebertozzo I have reverse my code. My change wasn't working correctly.

File: assets/dev/scss/editor/panel/controls/_select.scss [hunk 2]
```diff
@@ -39,4 +39,14 @@
 			pointer-events: none;
 		}
 	}
+
+	.elementor-control-field {
```
Comment (louiswol94) [inline assets/dev/scss/editor/panel/controls/_select.scss:43]: @hein-obox Check nesting.

```
.elementor-control {
    
    &-input-wrapper {}
    
    &-control-field {}
}
```

Older CSS and your new CSS can be nested inside `.elementor-control{}`

Also, this nesting is not necessary:

```
&.elementor-control-field-select-small {

	.elementor-control-input-wrapper {
		max-width: 80px;
	}
}
```

Change to:

```
&.elementor-control-field-select-small .elementor-control-input-wrapper {
	max-width: 80px;
}
```

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/controls/_select.scss (hunk 2 above)
Comment (hein-obox) [reply assets/dev/scss/editor/panel/controls/_select.scss]: Refactoring implemented.

@rodolphebertozzo Please check this out :)

**Review by rodolphebertozzo — DISMISSED** [review]
@hein-obox Looks good to me ;)

**Review by louiswol94 — DISMISSED** [review]

**Review by udidol — APPROVED** [review]

**Review by udidol — DISMISSED** [review]

File: tests/playwright/sanity/includes/widgets/image-carousel.test.js [hunk 3]
```diff
@@ -19,33 +19,49 @@ test( 'Image Carousel', async ( { page }, testInfo ) => {
...
 	await page.selectOption( '.elementor-control-navigation >> select', 'none' );
 
 	// Populate the widget with images.
 	await editor.populateImageCarousel();
 
-	expect( await editor.getPreviewFrame().locator( 'div.elementor-image-carousel-wrapper.swiper-container.swiper-container-initialized' ).screenshot( { type: 'jpeg', quality: 70 } ) ).toMatchSnapshot( 'carousel.jpeg' );
+	await test.step( 'Verify image population', async () => {
+		expect( await editor.getPreviewFrame().locator( 'div.elementor-image-carousel-wrapper.swiper-container.swiper-container-initialized' ).screenshot( {
+			type: 'jpeg',
+			quality: 70,
```
Comment (udidol) [inline tests/playwright/sanity/includes/widgets/image-carousel.test.js:33]:
```suggestion
			quality: 90,
```

**Review by udidol — APPROVED** [review]

**Pull request merged**